Apps/frontend/src/components/WaveControls.svelte

```diff
@@ -7,10 +7,48 @@
   import { timeSweep } from "../stores.js";
   import TimeSweepSlider from "./TimeSweepSlider.svelte";
   import ThicknessSwitch from "./ThicknessSwitch.svelte";
+  import DistributeOffsetButton from "./DistributeOffsetButton.svelte";
+  import startStopLine from "./Waves.svelte";
   export let waveElement;
   export let isEnabled;
   export let indicatorElement;
   let btnOnOff;
+
+  let activeChannels = [];
+  let activeChannelCounter = NUM_CHANNELS;
+
+  let distributed = false;
+
+  const setActiveChannels = () => {
+    for (let i = 0; i < NUM_CHANNELS; i++) {
+      activeChannels[i] = false;
+    }
+  };
+
+  setActiveChannels();
```

Comment:
If you want to explicitly set the active/disable all channels (instead of using a writable store), I think this should happen in `onMount`

Reply:
Also removed